gps2val/gpsd_feeder.py

```diff
@@ -54,11 +54,11 @@ def shutdown(self):
         self.client.stop()
 
     def setPosition(self, position):
-        self.client.setValue('Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Altitude', position['alt'])
-        self.client.setValue('Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Latitude', position["lat"])
-        self.client.setValue('Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Longitude', position["lon"])
-        self.client.setValue('Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Accuracy', position["hdop"])
-        self.client.setValue('Vehicle.Cabin.Infotainment.Navigation.CurrentLocation.Speed', position["speed"])
+        self.client.setValue('Vehicle.CurrentLocation.Altitude', str(position['alt']))
```

Comment:
For me running gps2val failed without adding `str` when using gpsfake and the included log file as input, it was complaining that `float`was not supported. I have tested with `str` towards kuksa-val-server and it works as expected.

@SebastianSchildt - do we know of anyone using this file that could help verifying this change.

Reply:
Ah, that is looking fine. Has to do with some typing assumptions that changed in the python SDK.